oracle/src/sponge.rs

```diff
@@ -60,8 +60,8 @@ impl<F: PrimeField> ScalarChallenge<F> {
 
 #[derive(Clone)]
 pub struct DefaultFqSponge<P: SWModelParameters, SC: SpongeConstants> {
-    pub sponge: ArithmeticSponge<P::BaseField, SC>,
-    pub last_squeezed: Vec<u64>,
+    sponge: ArithmeticSponge<P::BaseField, SC>,
```

Comment:
This one has a constructor `new`, and these fields are only needed at creation time so they seem to be able to remain private